src/d3a/models/market/two_sided_pay_as_bid.py

```diff
@@ -69,11 +75,16 @@ def delete_bid(self, bid_or_id: Union[str, Bid]):
         log.info(f"[BID][DEL][{self.time_slot_str}] {bid}")
         self._notify_listeners(MarketEvent.BID_DELETED, bid=bid)
 
+    def _update_fee_and_calculate_final_price(self, energy, trade_rate,
```

Comment:
How about being consistent with the `_update_offer_fee_and_calculate_final_price` in the one-sided-market and renaming it to `_update_bid_fee_and_calculate_final_price`

Reply:
Agreed. Renaming, thanks